README.md

```diff
@@ -1,24 +1,101 @@
-# README
+# Budget App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> This is a A rails application that allows users to create expense categories and track the transactions per category. A transaction can belong to multiple categories. It includes user authentication and data is preserved using postgresql.
 
-Things you may want to cover:
+## Built With
 
-* Ruby version
+- Ruby on Rails
+- postgresql
 
-* System dependencies
+## Live demo
 
-* Configuration
+[Live link](Coming soon...)
```

Comment:
- [ ] As is a [requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md)for  this project, " `Technical requirements: The project should be deployed and accessible online.` " you should deploy your application using Heroku, Netlify or GitHub Pages, make sure your project is deployed properly, the home and detail page works as expected, a  link to the deployed project should be present in the README.

Reply:
please check again, @t-yanick 
I added the live link